test/martha/martha_v3.test.js

```diff
@@ -466,10 +466,17 @@ test('should parse Data Object uri with jade data repo DEV as host', (t) => {
     );
 });
 
-test('should parse Data Object uri with jade data repo DEV as host and path with snapshot id', (t) => {
+test('should parse Data Object uri with jade data repo STAGING as host', (t) => {
     t.is(
-        determineDrsTypeTestWrapper('drs://jade.datarepo-dev.broadinstitute.org/v1_c78919df-5d71-414b-ad29-7c3c0d810657_973b5e79-6433-40ce-bf38-686ab7f17820'),
-        'https://jade.datarepo-dev.broadinstitute.org/ga4gh/drs/v1/objects/v1_c78919df-5d71-414b-ad29-7c3c0d810657_973b5e79-6433-40ce-bf38-686ab7f17820'
+        determineDrsTypeTestWrapper('drs://data.staging.envs-terra.bio/v1_anything'),
+        'https://data.staging.envs-terra.bio/ga4gh/drs/v1/objects/v1_anything'
+    );
+});
+
+test('should parse Data Object uri with jade data repo ALPHA as host', (t) => {
+    t.is(
+        determineDrsTypeTestWrapper('drs://data.alpha.envs-terra.bio/v1_anything'),
+        'https://data.alpha.envs-terra.bio/ga4gh/drs/v1/objects/v1_anything'
```

Comment:
Similar comment on other non-prod and non-dev hostnames out there that could be included OR more simply compressed into one test instead of N-today-plus-future-PRs.